src/pages/DatasetSearch.js

```diff
@@ -3,10 +3,43 @@ import { useEffect, useState } from 'react';
 import { Notifications } from '../libs/utils';
 import { DataSet } from '../libs/ajax';
 import DatasetSearchTable from '../components/data_search/DatasetSearchTable';
+import broadIcon from '../logo.svg';
+import duosIcon from '../images/duos-network-logo.svg';
+import mgbIcon from '../images/mass-general-brigham-logo.svg';
 
 export const DatasetSearch = (props) => {
+  const { location } = props;
   const [datasets, setDatasets] = useState([]);
 
+  // branded study table versions
+  const versions = {
+    '/datalibrary': {
+      query: null,
+      icon: duosIcon,
+      title: 'DUOS Data Library',
+    },
+    '/datalibrary_broad': {
+      query: {
+        'match': {
+          'submitter.institution.id': '150' // Broad Institute of MIT and Harvard
```

Comment:
Perhaps I should match on exact name instead since the ids might be different between environments.

Reply:
Yes, the IDs are different across environments, let's go with name